handsontable/src/plugins/contextMenu/menu.js

```diff
@@ -33,11 +33,26 @@ import localHooks from '../../mixins/localHooks';
 
 const MIN_WIDTH = 215;
 
+/**
+ * @typedef MenuOptions
+ * @property {Menu} [parent] Default value: `null`.
+ * @property {string} [name] Default value: `null`.
+ * @property {string} [className] Default value: `''`.
+ * @property {boolean} [keepInViewport] Default value: `true`.
+ * @property {boolean} [standalone] Default value: `false`.
+ * @property {number} [minWidth] Default value: `MIN_WIDTH`.
```

Comment:
I would not describe default value in this case. JSDoc won't accept variable.

Reply:
@wszymanski could you please propose descriptions? I have no experience with menu options and I don't know what they do.

Reply:
I don't now every part of API. Please do it by yourself 🙂